recipes/stats_can/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "stats_can" %}
+{% set version = "2.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6ed800301840682a32ee98a9963118d4fb1d65f49abe5a0f73173c8fd4301dba
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+    - python >=3.0
+    - pip
```

Comment:
You might need `poetry` as a dep here?

Reply:
I gave that a shot in build, but I'll try again in host. I have to admit I don't have a good mental model about the distinction between those two categories.

Reply:
Yup, putting it in host fixed it >.<
Guess I should have fiddled with one more thing before tagging in for help.
I appreciate the reply!